frappe/email/doctype/standard_reply/standard_reply.py

```diff
@@ -17,6 +17,10 @@ def get_standard_reply(template_name, doc):
 	if isinstance(doc, string_types):
 		doc = json.loads(doc)
 
+	the_doc = frappe.get_doc(doc.get('doctype'), doc.get('name'))
```

Comment:
better idea to just replace `doc` the `frappe.get_doc` ?

Reply:
You mean where we have the json.loads?